ts/types/wallet.ts

```diff
@@ -33,3 +35,12 @@ export const UNKNOWN_TRANSACTION: WalletTransaction = {
   transactionCost: 0,
   isNew: false
 };
+
+/**
+ * This type represents a transaction Manager. One manager should be associated to each payment method not handled by PagoPA
+ */
+export type TransactionManager = {
```

Comment:
wrap with `Readonly<>`

Reply:
lint doesn't like it: 
'TransactionManager' is declared but its value is never read

Reply:
That sounds unlikely. The fact that you're exporting the type should never trigger that error. Just so we're sure, you tried: 
```typescript
export type TransactionManager = Readonly<{
  id: number;
  maxFee: number;
  icon: ImageSource;
}>;
```
Correct?